src/pages/my_page/user_profile/MyProjects.tsx

```diff
@@ -0,0 +1,48 @@
+import { List, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { CristinApiPath } from '../../../api/apiPaths';
+import { ListSkeleton } from '../../../components/ListSkeleton';
+import { RootState } from '../../../redux/store';
+import { SearchResponse } from '../../../types/common.types';
+import { CristinProject } from '../../../types/project.types';
+import { useFetch } from '../../../utils/hooks/useFetch';
+import { CristinSearchPagination } from '../../search/CristinSearchPagination';
+import { ProjectListItem } from '../../search/project_search/ProjectListItem';
+
+export const MyProjects = () => {
+  const { t } = useTranslation();
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const user = useSelector((store: RootState) => store.user)!; // If user has been empty this route would already be blocked
+  const userCristinId = user.cristinId?.split('/').pop();
+
+  const queryParams = `query=.&participant=${userCristinId}`;
+
+  const [projectsSearch, isLoadingProjectsSearch] = useFetch<SearchResponse<CristinProject>>({
+    url: queryParams ? `${CristinApiPath.Project}?${queryParams}` : '',
```

Comment:
`queryParams` vil alltid ha en verdi og dermed bli true her, så kanskje endre til noe slik?
```suggestion
    url: userCristinId ? `${CristinApiPath.Project}?query=.&participant=${userCristinId}` : '',
```